canonical-websites/www.ubuntu.com
# Remove beta site feedback button

## Done

- removing the Site feedback mechanism as the beta has ended

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that there is nothing remaining from the Site feedback

## Issue / Card

Fixes #4238
